packages/schedule/src/__tests__/ScheduleDOM-test.js

```diff
@@ -309,10 +309,10 @@ describe('ScheduleDOM', () => {
           const callbackC = jest.fn(() => callbackLog.push('C'));
           const callbackD = jest.fn(() => callbackLog.push('D'));
 
-          scheduleWork(callbackA); // won't time out
+          scheduleWork(callbackA, {timeout: 100}); // won't time out
```

Comment:
Needed to make a small tweak to an existing test: Callbacks are now executed in timeout-order. This is the only semantic change.